tests/test_install.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+import sys
+
+import pytest
+
+import audobject
+
+
+yaml_with_version = '''
+$opensmile.core.smile.Smile==2.4.1:
```

Comment:
I also had to remove the test for Windows because of  `opensmile`, it had problems uninstalling the package (which we have to do during tests) because of a non-windows confirm path I think. Anyway, I will put WIP for the moment until we have switched to another package.

Reply:
Another option would be to switch to `audonnx`....